packages/react-topology/src/Visualization.ts

```diff
@@ -28,6 +29,25 @@ export class Visualization extends Stateful implements Controller {
   @observable.ref
   private graph?: Graph;
 
+  @observable
+  private viewConstraintsEnabled: boolean = true;
+
+  @observable.ref
+  private viewPaddingSettings: ViewPaddingSettings = {
+    paddingPercentage: 50
+  };
+
+  @computed
+  private get viewPadding(): number {
+    if (this.viewPaddingSettings.paddingPercentage) {
```

Comment:
nit (to reduce observable touches)
```suggestion
    const { padding, paddingPercentage } = this.viewPaddingSettings;
    if (paddingPercentage) {
```